tests/test_fixtures/rule_based_profiler/example_notebooks/MultiBatchExample_ConfiguredAssetSQLExample_SQL.ipynb

```diff
@@ -379,7 +367,9 @@
    "id": "ee01a0e9",
    "metadata": {},
    "source": [
-    "When you run methods on the validator, it will typically run on the most recent batch (index `-1`), even if the Validator has access to a longer Batch list. For example, notice that rows below are all associated with `pickup_datetime` being `9` (September, 2020). This is because the datetime values are stored lexicographically, meaning `1` and `11`, `12` values will appear **before** `2` and `3`."
+    "When you run methods on the validator, it will typically run on the most recent batch (index `-1`), even if the Validator has access to a longer Batch list. For example, notice that rows below are all associated with `pickup_datetime` being `9` (September, 2020). This is because the datetime values are stored lexicographically, meaning `1` and `11`, `12` values will appear **before** `2` and `3`.\n",
+    "\n",
+    "**NOTE** This will be fixed in time for the `DataAssistant` release"
    ]
   },
   {
```

Comment:
I made some suggestions to wording in some of these sections in https://github.com/great-expectations/great_expectations/pull/5958 that might be good to incorporate in this notebook as well.